docs/developer-docs/setup/hello-world.mdx

```diff
@@ -0,0 +1,254 @@
+```
+
+### Install Node.js
+
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+nvm install 18
+```
+
+## Start the local replica
+
+```bash
+dfx start
+```
+
+## Create a new project
+
+<Tabs groupId="language">
+<TabItem value="motoko" label="Motoko" default>
+
+```bash
+dfx new hello
+cd hello
+npm install
+```
+
+</TabItem>
+<TabItem value="rust" label="Rust">
+
+```bash
+dfx new hello --type=rust
+cd hello
+npm install
+```
+
+</TabItem>
+
+<TabItem value="typescript" label="TypeScript">
+
+```bash
+npx azle new azle_hello
+cd hello
+npm install
+```
+
+</TabItem>
+
+<TabItem value="python" label="Python">
+
+Create the project directory and file structure:
+
+```bash
+mkdir kybra_hello
+cd kybra_hello
+
+mkdir src
+
+touch src/main.py
+touch dfx.json
+```
+
+Create a virtual environment:
+
+```bash
+~/.pyenv/versions/3.10.7/bin/python -m venv venv
+source venv/bin/activate
+```
+
+Install Kybra:
+
+```bash
+pip install kybra
+```
+
+</TabItem>
+</Tabs>
+
+For information about project structure and default configuration, [learn more about deploying your first canister](first-canister.mdx).
+ 
+## Smart contract code
+
+<Tabs groupId="language">
+<TabItem value="motoko" label="Motoko" default>
+
+```motoko
+// src/hello_backend/main.mo
+actor {
+public query func greet(name : Text) : async Text {
+    return "Hello, " # name # "!";
+};
+};
+```
+
+</TabItem>
+<TabItem value="rust" label="Rust">
+
+```rust
+// src/hello_world_backend/src/lib.rs
+#[ic_cdk::query]
+fn greet(name: String) -> String {
+    format!("Hello, {}!", name)
+}
+```
+
+</TabItem>
+
+<TabItem value="typescript" label="TypeScript">
+
+```typescript
+// azle_hello/src/index.ts
+import { Canister, query, text, update, Void } from 'azle';
+
+// This is a global variable that is stored on the heap
+let message = '';
+
+export default Canister({
+    // Query calls complete quickly because they do not go through consensus
+    getMessage: query([], text, () => {
+        return message;
+    }),
+    // Update calls take a few seconds to complete
+    // This is because they persist state changes and go through consensus
+    setMessage: update([text], Void, (newMessage) => {
+        message = newMessage; // This change will be persisted
+    })
+});
+```
+
+</TabItem>
+
+<TabItem value="python" label="Python">
+
+Create the project directory and file structure:
+
+```python
+# kybra_hello/src/main.py
+
+from kybra import query, update, void
+
+# This is a global variable that is stored on the heap
+message: str = ''
+
+# Query calls complete quickly because they do not go through consensus
+@query
+def get_message() -> str:
+    return message
+
+# Update calls take a few seconds to complete
+# This is because they persist state changes and go through consensus
+@update
+def set_message(new_message: str) -> void:
+    global message
+    message = new_message # This change will be persisted
+```
+
+</TabItem>
+</Tabs>
+
```

Comment:
Let's add a section explaining the developer account

## Developer account

In order to deploy smart contracts, you need a developer account since that account will become the owner of the deployed smart contract. A developer account is just a pair of private and public keys. The address of the developer account is derived from its public key and is referred to as _principal_.

Your `dfx` has already generated a default developer account for you. You can get its principal with:
```bash
dfx identity get-principal
```

The principal will looks something like this:
```bash
itk7v-ihlxk-ktdrh-fcnst-vkoou-orj77-52ogl-jqwj5-zpfdv-az3lr-xqe
```

You can also inspect the corresponding private key by running:
```bash
dfx identity export default
```
The result may look something like this:
```
-----BEGIN EC PRIVATE KEY-----
MHQCAQEEIIGTvo7+8q+dRVoDWPVp1hBziKUEOffo1PDkY2Vn8sKMoAcGBSuBBAAK
oUQDQgAEXFFBa0oxGdamv537nusjtgVuzElEMtEbbj4dQ1xyu2WMa+PU9q9LIK26
Ar821EwGwp1Qc9Vv93FGPTvRb4HVXQ==
-----END EC PRIVATE KEY-----
```

You can generate a new developer account with
```
dfx identity new <name>
```
and switch to it with
```
dfx identity use <name>
```

Reply:
Linking Dev Account doc to prevent duplication